src/main/java/com/ticketbooking/model/Reservation.java

```diff
@@ -0,0 +1,78 @@
+package com.ticketbooking.model;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name="reservations")
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String userName;
+    private String userSurname;
+    private LocalDateTime reservationTime;
+    private Boolean confirmed;
+
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
dlaczego lazy? jakie to może spowodować problemy?

Reply:
Z eager możemy przez przypadek spowodowac pobranie całej bazy danych w jednym zapytaniu, a lazy nas przed tym zabezpiecza.